cmd/exec/main.go

```diff
@@ -698,6 +706,39 @@ var deleteFilterCmd = &cobra.Command{
 	},
 }
 
+var getFilterCmd = &cobra.Command{
+	Use:   "get-filter NAME",
+	Short: "Get an event filter.",
+	Args:  cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+
+		filterName := args[0]
+
+		resp, err := executeGetCloudEventFilter(filterName)
+		if err != nil {
+			log.Fatalf("error: %v\n", err)
+		}
+
+		cmd.PrintErrln(string(resp))
```

Comment:
This command outputs the response as JSON. This should be an option, but I think human-readable should be the default. 

So let's add a flag that can set the output format. We could copy the `kubectl` flag here. Type `kubectl help get` and look at the bottom: `[(-o|--output=)json|yaml|name|go-template|go-template-file|template|templatefile|jsonpath|jsonpath-as-json|jsonpath-file|custom-columns|custom-columns-file|wide]`. We don't need to start with a long list of formats. Just default (human readable) and JSON for now.